### PR TITLE
It allows greater control of php packages installed on the nodes

### PR DESCRIPTION
**Pull request by thiagotalma**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -9,7 +9,22 @@ php5_socket_listen_mode: '0660'
 php5_memory_limit: '128M'
 
 # Install set of standard packages
-php5_packages: [ 'php5-curl', 'php5-gd', 'php5-mcrypt' ]
+php5_packages_default: [ 'php5-curl', 'php5-gd', 'php5-mcrypt' ]
```
Comment (drybjed) [inline defaults/main.yml]: I would go with `php5_base_packages`, to keep the theme of `php5_*_packages`. Would you like to rename this variable?

**Comment by drybjed** [comment]
Good idea, thanks!

File: defaults/main.yml (hunk 1 above)
Comment (thiagotalma) [inline defaults/main.yml]: @drybjed 
Done!

**Comment by drybjed** [comment]
@thiagotalma Awesome! I'm gonna merge that probably tommorow, since I plan to do signed merges from now on. Thanks for the PR. :-)

**Pull request merged**